x/stakeibc/keeper/redemption_sweep.go

```diff
@@ -49,11 +49,6 @@ func (k Keeper) SweepAllUnbondedTokensForHostZone(ctx sdk.Context, hostZone type
 		if inTransferQueue && validUnbondingTime {
 			k.Logger(ctx).Info(utils.LogWithHostZone(hostZone.ChainId, "  %v%s included in sweep", hostZoneUnbonding.NativeTokenAmount, hostZoneUnbonding.Denom))
 
-			if err != nil {
```

Comment:
why remove this?

Reply:
the `err` variable in this statement is actually defined way up above. This was caught by the linter btw

My guess is that we used to have something like
```go
if inTransferQueue && validUnbondingTime {
    k.Logger(...)
    
    err := something...  <---
    if err != nil {
        ...
```

and we removed the line with the arrow, but not the corresponding if statement below